config.js

```diff
@@ -585,41 +585,56 @@ var config = {
     //     alwaysVisible: false
     // },
 
-    // Toolbar buttons which have their click event exposed through the API on
-    // `toolbarButtonClicked` event instead of executing the normal click routine.
+    // Toolbar buttons which have their click/tap event exposed through the API on
+    // `toolbarButtonClicked`. Passing a string for the button key will
+    // prevent execution of the click/tap routine; passing an object with `key` and
```

Comment:
passing just a string should not prevent execution right? for backwards compatibility